content/influxdb/cloud-iox/get-started/_index.md

```diff
@@ -0,0 +1,126 @@
+with widely used data processing tools and platforms.
+
+**Time series data** is a sequence of data points indexed in time order.
+Data points typically consist of successive measurements made from the same
+source and are used to track changes over time.
+Examples of time series data include:
+
+- Industrial sensor data
+- Server performance metrics
+- Heartbeats per minute
+- Electrical activity in the brain
+- Rainfall measurements
+- Stock prices
+
+This multi-part tutorial walks you through writing time series data to InfluxDB {{< current-version >}},
+querying, and then visualizing that data.
+
+## Key concepts before you get started
+
+Before you get started using InfluxDB, it's important to understand how time series
+data is organized and stored in InfluxDB and some key definitions that are used
+throughout this documentation.
+
+- [Data organization](#data-organization)
+- [Schema on write](#schema-on-write)
+- [Important definitions](#important-definitions)
+
+### Data organization
+
+The InfluxDB data model organizes time series data into buckets and measurements.
+A bucket can contain multiple measurements. Measurements contain multiple
+tags and fields.
+
+- **Bucket**: Named location where time series data is stored.
+  A bucket can contain multiple _measurements_.
+  - **Measurement**: Logical grouping for time series data.
+    All _points_ in a given measurement should have the same _tags_.
+    A measurement contains multiple _tags_ and _fields_.
+      - **Tags**: Key-value pairs that provide provide metadata for each point--for example,
+        something to identify the source of the data like host, location, station, etc.
+      - **Fields**: Key-value pairs with values that change over time--for example: temperature, pressure, stock price, etc.
+      - **Timestamp**: Timestamp associated with the data.
+        When stored on disk and queried, all data is ordered by time.
+
+<!-- _For detailed information and examples of the InfluxDB data model, see
+[Data elements](/influxdb/v2.5/reference/key-concepts/data-elements/)._ -->
+
+### Schema on write
+
+When using InfluxDB, you define your schema as you write your data.
+You don't need to create measurements (equivalent to a relational table) or
+explicitly define the schema of the measurement.
+Measurement schemas are defined by the schema of data as it is written to the measurement.
+
+### Important definitions
+
+The following are important definitions to understand when using InfluxDB:
+
+- **Point**: Single data record identified by its _measurement, tag keys, tag values, field key, and timestamp_.
+- **Series**: A group of points with the same _measurement, tag keys and values, and field key_.
+- **Primary key**: Columns used to uniquely identify each row in a table.
+  Rows are uniquely identified by their _timestamp and tag set_.
+
+##### Example InfluxDB query results
+
+{{< influxdb/points-series-sql >}}
```

Comment:
the table in localhost has some strange underlining

Reply:
The underlines are to show how each row contains multiple points. If you hover over them, a key appears:

<img width="708" alt="image" src="https://user-images.githubusercontent.com/181484/210599041-b6690ea1-2a58-40c5-a917-838190ad47c4.png">